viam_cartographer.go

```diff
@@ -127,9 +127,11 @@ func initSensorProcess(cancelCtx context.Context, cartoSvc *CartographerService)
 	}
 
 	cartoSvc.sensorProcessWorkers.Add(1)
+	isOnline := cartoSvc.lidarDataRateMsec != 0
```

Comment:
can we remove the changes to this file?

Reply:
fixed thanks for catching! i am surprised i didn't get an error for isOnline not being used in Start